gems/flipper/0.28/_test/Steepfile

```diff
@@ -0,0 +1,12 @@
+D = Steep::Diagnostic
+
+target :test do
+  check "."
+  signature "."
+
+  repo_path "../../../"
+  library "flipper"
+  library "forwardable"
```

Comment:
This should be added to the `manifest.yaml`. At this moment, the following error was raised for me.

```
rbs -Isig validate --silent
/usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/errors.rb:239:in `check!': /app/.gem_rbs_collection/flipper/0.28/flipper.rbs:6:2...6:20: Could not find mixin: Forwardable (RBS::NoMixinFoundError)
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:375:in `block in mixin_ancestors0'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/ast/declarations.rb:48:in `each'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/ast/declarations.rb:48:in `each_mixin'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:338:in `mixin_ancestors0'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:397:in `block in mixin_ancestors'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:389:in `each'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:389:in `mixin_ancestors'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:307:in `one_singleton_ancestors'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder/ancestor_builder.rb:492:in `singleton_ancestors'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder.rb:295:in `block in build_singleton'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder.rb:748:in `try_cache'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/definition_builder.rb:291:in `build_singleton'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/cli.rb:470:in `block in run_validate'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/cli.rb:465:in `each'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/cli.rb:465:in `run_validate'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/lib/rbs/cli.rb:137:in `run'
        from /usr/local/bundle/bundler/gems/rbs-88b18802aa9e/exe/rbs:7:in `<top (required)>'
        from /usr/local/bundle/bin/rbs:25:in `load'
        from /usr/local/bundle/bin/rbs:25:in `<main>'
```

Reply:
I just posted https://github.com/ruby/gem_rbs_collection/pull/391 to fix this problem.

Thank you for your work. This PR removes local types from my environment. This must be helpful!